CSharp/Library/Microsoft.Bot.Connector.Teams.Shared/ConversationExtensions.cs

```diff
@@ -88,5 +193,69 @@ public static IEnumerable<TeamsChannelAccount> AsTeamsChannelAccounts(this IEnum
                 yield return JObject.FromObject(channelAccount).ToObject<TeamsChannelAccount>();
             }
         }
+
+        private static async Task<Tuple<HttpRequestMessage, HttpResponseMessage>> SetupAndSendHttpRequestGetConversationMember(ConnectorClient Client, string conversationId, string userId, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
can we please further split this method to "SetupConversationMemberRequest" and "GetConversationMember"